models/Show.js

```diff
@@ -0,0 +1,39 @@
+const {Model,DataTypes}=require('sequelize');
+const sequelize=require('../config/connection');
+
+class Show extends Model{}
+
+Model.init(
+    {
+        id:{
+            type:DataTypes.INTEGER,
+            allowNull:false,
+            primaryKey:true,
+            autoIncrement:true,
+        },
+        name:{
+            type:DataTypes.STRING,
+            allowNull:false,
+        },
+        premiereDate:{
+            type:DataTypes.DATE,
```

Comment:
I used DATEONLY for this instead of DATE, because it's a slightly better looking date to interact with, but we can work with this if needed. Probably just need to figure out which one and then how we want to handle prettying up the date we display to users.